CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 # 0.6.0 (unreleased)
 
 * When running with PHP 8, process attributes in addition to the phpdoc annotations.
+* Support doctrine collections
+* Support identical naming strategy
```

Comment:
```suggestion
* Support `identical` property naming strategy
```

Reply:
Thank you - I amended it to the last commit